### ISSUE TITLE
Type error preventing app from deploying 'No inputs were found in config file '.../node_modules/react-number-format/tsconfig.json'.  

### ISSUE THREAD

**Issue by monecchi** [s-yadav/react-number-format#840]
#### The following type error is preventing my project from being deployed:

```cmd
No inputs were found in config file '.../node_modules/react-number-format/tsconfig.json'. Specified 'include' paths were '["src","test/types"]' and 'exclude' paths were '[]'.
```
I know I can supress that kind of warning during deployment, but the same type error persists under development as typescript keeps complaining about it indefinitely. 

#### It seems the package has no .ts files to compile
The issue could be solved by adding at least  one empty .ts file to it.

**Comment by s-yadav** [comment]
I think we can just exclude it during publish to npm. As project is already compiled to JS, including tsconfig is not necessary.

**Comment by s-yadav** [comment]
this is fixed on v5.4.0
